website/docs/cloud-docs/vcs/bitbucket-data-center.mdx

```diff
@@ -0,0 +1,121 @@
+
+   If this results in a 500 error, it usually means Terraform Cloud was unable to reach your Bitbucket Data Center instance.
+
+1. Click the "Allow" button. This returns you to Terraform Cloud to enter a SSH key.
+
+## Step 3: On Workstation: Create an SSH Key for Terraform Cloud
+
+On a secure workstation, create an SSH keypair that Terraform Cloud can use to connect to Bitbucket Data Center. The exact command depends on your OS, but is usually something like `ssh-keygen -t rsa -m PEM -f "/Users/<NAME>/.ssh/service_terraform" -C "service_terraform_enterprise"`. This creates a `service_terraform` file with the private key, and a `service_terraform.pub` file with the public key.
+
+This SSH key **must have an empty passphrase.** Terraform Cloud cannot use SSH keys that require a passphrase.
+
+### Important Notes
+
+- Do not use your personal SSH key to connect Terraform Cloud and Bitbucket Data Center; generate a new one or use an existing key reserved for service access.
+- In the following steps, you must provide Terraform Cloud with the private key. Although Terraform Cloud does not display the text of the key to users after it is entered, it retains it and will use it for authenticating to Bitbucket Data Center.
+- **Protect this private key carefully.** It can push code to the repositories you use to manage your infrastructure. Take note of your organization's policies for protecting important credentials and be sure to follow them.
+
+## Step 4: On Bitbucket Data Center, Switch Users and Add an SSH Key
+
+1. If you are still logged in to Bitbucket Data Center as an administrator, log out now.
+
+1. Log in as whichever account you want Terraform Cloud to act as. For most organizations this should be a dedicated service user, but a personal account will also work.
+
+   ~> **Important:** The account you use for connecting Terraform Cloud **must have admin access** to any shared repositories of Terraform configurations, since creating webhooks requires admin permissions.
+
+1. Go to the "SSH keys" page. You can click the profile icon, choose "Manage account". Click "SSH keys" or you can go directly to `https://<BITBUCKET INSTANCE HOSTNAME>/plugins/servlet/ssh/account/keys`.
+
+   ![Bitbucket Data Center screenshot: the SSH keys page](/img/docs/bitbucket-server-ssh-keys.png)
+
+1. Click the "Add key" button. Paste the text of the **SSH public key** you created in step 4 (from the `.pub` file) into the text field, then click the "Add key" button to confirm.
+
+## Step 5: On Terraform Cloud, Request Access and Add an SSH Private Key
+
+1. Click the "Add a private SSH key" link. A large text field will appear. Paste the text of the **SSH private key** you created in step 3, and click the "Add SSH Key" button.
+
+## Finished
+
+At this point, Bitbucket Data Center access for Terraform Cloud is fully configured, and you can create Terraform workspaces based on your organization's shared repositories.
```

Comment:
```suggestion
## Add an SSH Private Key

Complete the following steps in Terraform Cloud or Terraform Enterprise to request access to Bitbucket and dd the SSH private key.

1. Open the **SSH keys** settings page and click **Add a private SSH key**. A large text field appears. 
1. Enter the text of the **SSH private key** you created in [Create an SSH key for Terraform](#create-an-ssh-key-for-terraform) and click **Add SSH Key**.

## Next steps

After completing these instructions, you can create Terraform workspaces based on your organization's shared repositories. Refer to the following resources for additional guidance:

- [Creating Workspaces](/terraform/cloud-docs/workspaces/creating) in Terraform Cloud
- [Creating Workspaces](/terraform/enterprise/workspaces/creating) in Terraform Enterprise
```
The standardized format includes a section we're calling Next steps to link to documentation that users might want to follow next.

Reply:
This is step 5 in a [5 step series](https://terraform-docs-common-git-paul-tf-12000-hashicorp.vercel.app/terraform/cloud-docs/vcs/bitbucket-data-center#step-5-on-terraform-cloud-request-access-and-add-an-ssh-private-key). Is there a reason to not include that in the section title?